src/recordlinker/schemas/pii.py

```diff
@@ -441,27 +438,31 @@ def feature_iter(self, feature: Feature) -> typing.Iterator[str]:
                     yield str(race)
         elif attribute == FeatureAttribute.TELECOM:
             for telecom in self.telecom:
+                value = telecom.value.strip()
```

Comment:
I don't think we need to call `strip()`, all whitespace should have been removed by the model_validator